package-lock.json

```diff
@@ -6,6 +6,7 @@
         "": {
             "name": "root",
             "dependencies": {
+                "@hashgraph/hedera-local": "^1.2.0",
```

Comment:
What changes were made in this file?
They weren't captured in the PR description